lpc8xx-hal-common/src/lib.rs

```diff
@@ -1,7 +1,40 @@
 #![no_std]
 
+use embedded_hal as hal;
 #[cfg(feature = "82x")]
-pub use lpc82x_pac as target;
+pub use lpc82x_pac as raw;
```

Comment:
I actually prefer `target` to `raw`, but don't feel strongly about it. Do as you think is best!

Reply:
I'll keep raw for now, so that the name is consistent between both crates